hyperspy/misc/eds/utils.py

```diff
@@ -89,22 +115,30 @@ def get_FWHM_at_Energy(energy_resolution_MnKa, E):
 
     Notes
     -----
-    From the textbook of Goldstein et al., Plenum publisher,
-    third edition p 315
+    This method implements the equation derived by Fiori and Newbury as is
+    documented in the following:
+
+        Fiorie, C. E., and Newbury, D. E. (1978). In SEM/1978/I, SEM, Inc.,
+        AFM O'Hare, Illinois, p. 401.
+
+        Goldstein et al. (2003). "Scanning Electron Microscopy & X-ray
+        Microanalysis", Plenum, third edition, p 315.
 
     """
     FWHM_ref = energy_resolution_MnKa
     E_ref = _get_energy_xray_line('Mn_Ka')
 
-    FWHM_e = 2.5 * (E - E_ref) * 1000 + FWHM_ref * FWHM_ref
+    FWHM_e = 2.5 * (E - E_ref) * eV2keV + FWHM_ref * FWHM_ref
 
-    return math.sqrt(FWHM_e) / 1000  # In mrad
+    return math.sqrt(FWHM_e) / 1000.  # In mrad
 
 
 def xray_range(xray_line, beam_energy, density='auto'):
-    """Return the Anderson-Hasler X-ray range.
+    """Return the maximum range of X-ray generation according to the
+    Anderson-Hasler parameterization.
 
-    Return the maximum range of X-ray generation in a pure bulk material.
+    Note that the magic numbers appearing in the equation are the values from
+    the parameterization mentioned above.
```

Comment:
In general, docstrings are how to use the function. I feel this belongs as a comment inside the function body, e.g. next to the first magic number. Same for `electron_range`.